frontend/Themes/Themes.js

```diff
@@ -224,6 +231,7 @@ const FirefoxFirebug: Theme = {
   special05: '#0000ff',
   special06: '#000062',
   special07: '#0000ff',
+  special08: '#0000ff',
```

Comment:
No this was intentional, this theme is incredibly color-poor (I assumed for accessibility) so I thought it would make sense to not differentiate HOCs on this one. Kind of weird I know.. but looking at the theme it made sense to me ?